nuxt.config.ts

```diff
@@ -173,6 +173,7 @@ const config: NuxtConfig = {
     '@nuxtjs/redirect-module',
     '@nuxtjs/sentry',
     '@nuxtjs/sitemap',
+    'cookie-universal-nuxt',
```

Comment:
It's a shame we're not on Nuxt3 and can't use `useCookie` yet :cry:

Reply:
+💯